# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/tests/browser.ini
# Commit: 1cc86398ced0
# Full Hash: 1cc86398ced034fafc66d94799f91221420e1229
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2021-02-18 04:16:06
# Regressor Bug: 1692350
# File Overlap Count: 1
# Description:
#   Bug 1692350 - Implement a telemetry probe to collect page load data for documents that use lazyload r=emilio,chutten
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104912
# ==============================================================================

diff -r 1ca4c3365fae -r 1cc86398ced0 layout/base/tests/browser.ini
--- a/layout/base/tests/browser.ini	Wed Feb 17 22:02:17 2021 +0000
+++ b/layout/base/tests/browser.ini	Wed Feb 17 22:25:02 2021 +0000
@@ -26,3 +26,7 @@
 support-files =
   file_lazyload_telemetry.html
   image_rgrg-256x256.png
+[browser_lazyload_page_load_telemetry_iframe.js]
+support-files =
+  file_lazyload_telemetry.html
+  image_rgrg-256x256.png